src/backend/app/db/crud.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import OperationalError
+import os
+
+DATABASE_URL = os.getenv("DATABASE_URL")
+
+engine = create_engine(DATABASE_URL)
```

Comment:
Why do we create a database connection in two different files? It seems like some of this is superfluous...

Reply:
right